lms/djangoapps/instructor/views/api.py

```diff
@@ -2205,16 +2204,10 @@ def list_report_downloads(_request, course_id):
     """
     List grade CSV files that are available for download for this course.
     """
-    course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
-    report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
-
-    response_payload = {
-        'downloads': [
-            dict(name=name, url=url, link='<a href="{}">{}</a>'.format(url, name))
-            for name, url in report_store.links_for(course_id)
-        ]
-    }
-    return JsonResponse(response_payload)
+    # to avoid circular imports
+    from instructor.views.gradebook_api import list_report_downloads_by_course_key
```

Comment:
Can you refactor these files so that you don't need to do an inline import? Maybe put this function in a different file that both `instructor/views/api.py` and `instructor/views/gradebook_api.py` both import?